addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/ProxyDisclosureScanRule.java

```diff
@@ -193,6 +194,11 @@ public void init() {
     @Override
     public void scan() {
         try {
+            if (StringUtils.containsIgnoreCase(
+                    getBaseMsg().getRequestHeader().getHeadersAsString(), "x-forward")) {
```

Comment:
This does not fix the mentioned issue, it should check the response body.

Reply:
Also, better to use the patterns of `PROXY_REQUEST_HEADERS`.

Reply:
Thanks, I misunderstood the original ticket. I thought they were saying that the ticket was reflecting something that was being requested. Not that it simply always responded with that body content. Will tweak.